clang/utils/TableGen/ClangProgModelBuiltinEmitter.cpp

```diff
@@ -848,6 +922,228 @@ void BuiltinNameEmitter::EmitQualTypeFinder() {
   OS << "\n} // Bultin2Qual\n";
 }
 
+JSONBuiltinInterfaceEmitter::TypeDesc::TypeDesc(const Record *T) : Rec(T) {
+  auto ToOCLAddressSpace = [](llvm::StringRef s) {
+    if (s == "clang::LangAS::Default")
+      return "";
+    if (s == "clang::LangAS::opencl_private")
+      return "__private";
+    if (s == "clang::LangAS::opencl_global")
+      return "__global";
+    if (s == "clang::LangAS::opencl_constant")
+      return "__constant";
+    if (s == "clang::LangAS::opencl_local")
+      return "__local";
+    return "__generic";
+  };
+
+  Name = T->getValueAsString("Name");
+  IsInteger = T->getValueAsBit("IsInteger");
+  IsSigned = T->getValueAsBit("IsSigned");
+  IsFloat = T->getValueAsBit("IsFloat");
+  Size = T->getValueAsInt("ElementSize");
+  VecWidth = T->getValueAsInt("VecWidth");
+  IsConst = T->getValueAsBit("IsConst");
+  IsVolatile = T->getValueAsBit("IsVolatile");
+  IsPointer = T->getValueAsBit("IsPointer");
+  IsBuiltin =
+      !T->isSubClassOf("FundamentalType") && !T->isSubClassOf("VectorType");
+
+  AddrSpace = ToOCLAddressSpace(T->getValueAsString("AddrSpace"));
+}
+
+std::string JSONBuiltinInterfaceEmitter::TypeDesc::GetBaseTypeAsStr() const {
+  if (Name == "void") {
```

Comment:
Don't use curleys for single line if/else/etc.  Applies through this patch.

Reply:
fixed

Reply:
Is it?  I see some below as well as here still.

Reply:
done, BTW I can't find where the coding style mention this.

Reply:
Apparently it is just something that has been enforced forever without having ever been put into the document!  I'm likely going to add it to the document to be consistent with what reviewers enforce.